luaui/Widgets/api_resource_spot_builder.lua

```diff
@@ -109,7 +109,7 @@ end
 -- Helper functions (Math stuff)
 ------------------------------------------------------------
 
-local function Distance(x1, z1, x2, z2)
+local function distance2dSquared(x1, z1, x2, z2)
 	return (x1 - x2) * (x1 - x2) + (z1 - z2) * (z1 - z2)
```

Comment:
A minor perf improvement if you care: https://springrts.com/wiki/Lua_Performance#TEST_6:_.27x.5E2.27_vs._.27x.2Ax.27